packages/server/src/fhir/repo.ts

```diff
@@ -669,7 +671,8 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
   private async updateResourceImpl<T extends Resource>(
     resource: T,
     create: boolean,
-    versionId?: string
+    versionId?: string,
+    inheritAccounts?: boolean
```

Comment:
Do we want to start a new `options` Object here, to accommodate any future parameters as well?